packages/browser-wallet/src/popup/store/utils.ts

```diff
@@ -68,6 +71,21 @@ export function atomWithChromeStorage<V>(key: ChromeStorageKey, fallback: V, wit
                 value: value ?? fallback,
             })
         );
+
+        if (withSync) {
+            const listener = (changes: Record<string, chrome.storage.StorageChange>) =>
+                getStoredValue().then((value) => {
+                    if (key in changes) {
+                        setValue({
+                            loading: false,
+                            value: value ?? fallback,
+                        });
+                    }
+                });
+            chrome.storage.local.onChanged.addListener(listener);
```

Comment:
`local` could also be any other chrome storage type, so it should be made more generic.

Reply:
Yes, I have fixed that now.